pkg/protocol/bthrift/unknown.go

```diff
@@ -36,363 +32,61 @@ type UnknownField struct {
 	Value   interface{}
 }
 
-// GetUnknownFields deserialize unknownFields stored in v to a list of *UnknownFields.
-func GetUnknownFields(v interface{}) (fields []UnknownField, err error) {
-	var buf []byte
-	rv := reflect.ValueOf(v)
-	if rv.Kind() == reflect.Ptr && !rv.IsNil() {
-		rv = rv.Elem()
+func fromGopkgUnknownFields(ff []unknownfields.UnknownField) []UnknownField {
```

Comment:
The reason why UnknownField does not directly alias is because the Name field wants to be removed? 
Name will not be used in codec, what was the initial consideration for giving the definition? @jayantxie